content/embeds/hardware-requirements-embed.md

```diff
@@ -48,18 +50,23 @@ Additional considerations:
 
     - If some of the cluster nodes are utilizing more than 80% of the CPU, consider migrating busy resources to less busy nodes.
 
-    - If all the cluster nodes are utilizing over 80% of the CPU, consider scaling out the cluster by [adding a node]({{< relref "/operate/rs/clusters/add-node" >}}).
+    - If all the cluster nodes are utilizing over 80% of the CPU, highly consider scaling out the cluster by [adding a node]({{< relref "/operate/rs/clusters/add-node" >}}).
 
 3. <a name="table-note-3"></a> RAM:
 
     - Redis uses a relatively large number of buffers, which enable replica communication, client communication, pub/sub commands, and more.  As a result, you should ensure that 30% of the RAM is available on each node at any given time.
 
     - If one or more cluster nodes utilizes more than 65% of the RAM, consider migrating resources to less active nodes.
 
-    - If all cluster nodes are utilizing more than 70% of available RAM, consider [adding a node]({{< relref "/operate/rs/clusters/add-node" >}}).
+    - If all cluster nodes are utilizing more than 70% of available RAM, highly consider [adding a node]({{< relref "/operate/rs/clusters/add-node" >}}).
 
     - Do not run any other memory-intensive processes on the Redis Enterprise Software node.
+  
+3. <a name="table-note-4"></a> RAM:
+
+    - Redis uses a relatively large number of buffers, which enable replica communication, client communication, pub/sub commands, and more.  As a result, you should ensure that 30% of the RAM is available on each node at any given time.
+
 
-4. <a name="table-note-4"></a>Network:
+4. <a name="table-note-4"></a>Persistent storage:
 
-    - Only static IP addresses are supported to ensure nodes remain part of the cluster after a reboot.
+    - When all databases on the cluster do not use, and will not enable [persistency]({{< relref "/operate/rs/installing-upgrading/install/plan-deployment/persistent-ephemeral-storage" >}}), minimal Persistent Storage can be lowered to RAM x 1.1 and recommended to RAM x 2. Having a Persistent Storage is essencial as it's being used also by Redis Enteprise to maintain the cluster and database health, configurations, recovery procedures and more.
```

Comment:
I was a little confused by the original wording, so I made a few suggested changes here, but please double-check that I haven't misunderstood something.
```suggestion
     - If no databases on the cluster have [persistence]({{< relref "/operate/rs/installing-upgrading/install/plan-deployment/persistent-ephemeral-storage" >}}) enabled, minimum persistent storage is RAM x 1.1 and the recommended persistent storage is RAM x 2. Persistent storage is essential because Redis Enterprise also uses it to maintain the cluster and database health, configurations, recovery procedures, and more.
```

Reply:
Also did you remove "Only static IP addresses are supported to ensure nodes remain part of the cluster after a reboot" on accident or was that intentional?

Reply:
ohh that an accident , not sure how this happened